islands/board.tsx

```diff
@@ -1,292 +1,331 @@
-import { useEffect, useState } from "https://esm.sh/v106/preact@10.11.0/hooks"
+import { useState } from "https://esm.sh/v106/preact@10.11.0/hooks"
 import Piece from "../components/piece.tsx"
 import { Square } from "../components/square.tsx"
+import {
+  rookCanMoveThere,
+  whitePawnCanMoveThere,
+} from "../scripts/pieceMoves.ts"
 
-type piece = '♟' | '♞' | '♝' | '♜' | '♛' | '♚' | '♙' | '♘' | '♗' | '♖' | '♕' | '♔' | ''
-type team = 'black' | 'white' | ''
+type piece =
+  | "♟"
```

Comment:
yeah, the formatting changes are bs. I prefer single quotes unless double is explicitly necessary